src/extension.ts

```diff
@@ -218,3 +226,38 @@ export function activate(context: vscode.ExtensionContext) {
 
 export function deactivate() {
 }
+
+function activateDps(context: vscode.ExtensionContext) {
+    let packageJSON = vscode.extensions.getExtension(Constants.ExtensionId).packageJSON;
+    let telemetryReporter = new TelemetryReporterWrapper(Constants.ExtensionId, packageJSON.version, packageJSON.aiKey, context, "DPS.");
+    let outputChannel = createAzExtOutputChannel("Device Provisioning Service", "azure-iot-toolkit");
+
+    // No need to initialize Constants again here. But pay attention when refactor.
+    // Constants.initialize(context);
+
+    let uiExtensionVariables = {
+        context: context,
+        outputChannel: outputChannel,
+        reporter: telemetryReporter,
+        ui: new AzureUserInput(context.globalState),
+    };
+    registerUIExtensionVariables(uiExtensionVariables);
+
+    let dpsTreeItem = new DpsAccountTreeItem();
+    let dpsExtTreeDataProvider = new AzExtTreeDataProvider(dpsTreeItem, "azure-iot-dps.loadMore");
+
+    context.subscriptions.push(dpsTreeItem);
```

Comment:
do we need to push dpsTreeItem here or it will be disposed with dpsExtTreeDataProvider?

Reply:
It won't be disposed with dpsExtTreeDataProvider. We need to dispose it separately.